SPECS/gpumon.spec

```diff
@@ -49,6 +49,11 @@ DESTDIR=%{buildroot} %{__make} install
 %{_unitdir}/xcp-rrdd-gpumon.service
 
 %changelog
+* Mon Nov 24 2025 Pau Ruiz Safont <pau.safont@vates.tech> - 24.1.0-71.1
+- Sync with 24.1.0-71
+- *** Upstream changelog ***
+  [only rebuilds]
```

Comment:
newbie question here:

Why aren't those lines preserved, as nested entries ?
* Tue Oct 14 2025 Gabriel Buica <danutgabriel.buica@cloud.com> - 24.1.0-71
- Bump release and rebuild
- (...)

Reply:
sure they are, but i would have kept them just to have a deterministic process (that could be automated) .. not a big deal anyway /i am approving it